docs/source/projects/architecture/content-architecture.rst

```diff
@@ -66,6 +67,7 @@ the core execution part. Native kernels implement execution and communications
 in the target language:
 
 .. image:: figs/other_kernels.png
+   :alt: 
```

Comment:
```suggestion
   :alt: Diagram showing how a wrapper kernel differs from a native kernel. Wrapper reuses IPython's message-passing code; native does not.
```

Reply:
Per Ely's suggestion, maybe we should set do blank alt-text: alt=""

Reply:
[https://www.w3.org/WAI/tutorials/images/decision-tree/](See W3C's alt Decision Tree)